staubli_resources/urdf/common_colours.xacro

```diff
@@ -2,9 +2,15 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
 
   <!-- RAL 1028 -->
-  <xacro:property name="colour_staubli_yellow" value="1.0 0.608 0 1" />
+  <xacro:property name="melon_yellow" value="1.0 0.608 0.0 1.0" />
```

Comment:
Please keep the `colour_` prefix. Normal xacro does not have the concept of namespaces (so it's easy to end up with name clashes) and it makes sense to explicitly name entities (ie: `colour_` for colours, `material_` for materials, etc).